plugins/rbac/src/hooks/useSelectedMembers.test.ts

```diff
@@ -0,0 +1,232 @@
+import { waitFor } from '@testing-library/react';
+import { renderHook } from '@testing-library/react-hooks';
+
+import { useSelectedMembers } from './useSelectedMembers';
+
+jest.mock('@backstage/core-plugin-api', () => ({
+  ...jest.requireActual('@backstage/core-plugin-api'),
+  useApi: jest.fn().mockReturnValue({
+    getRole: jest.fn().mockReturnValue([
+      {
+        memberReferences: [
+          'group:default/admins',
+          'user:default/amelia.park',
+          'user:default/calum.leavy',
+          'group:default/team-b',
+          'group:default/team-c',
+        ],
+        name: 'role:default/rbac_admin',
+      },
+    ]),
+    getMembers: jest.fn().mockReturnValue([
+      {
```

Comment:
nit: good to move the test data in `fixtures` so that can be used across if needed